command/runner.go

```diff
@@ -17,15 +17,21 @@ import (
 )
 
 var (
-	Stdout = colorable.NewColorableStdout()
-	Stderr = colorable.NewColorableStderr()
+	Stdout        = colorable.NewColorableStdout()
+	Stderr        = colorable.NewColorableStderr()
+	DefaultPagers = []string{"less", "more"}
 )
 
 type Printers struct {
 	FullColoredPrinter printer.Printer
 	ErrorPrinter       printer.Printer
 }
 
+type pagerPipe struct {
+	writer *os.File
+	cc     *chan struct{}
```

Comment:
These two don't need to be pointers, no?

Reply:
Indeed, no need